documentation/tutorials/get-started.md

```diff
@@ -385,28 +390,37 @@ tickets =
       ticket
     end
   end
+```
 
+Find the tickets where the subject contains `"2"`. Note that the we're setting the ticket data that we're querying using `set_data`.
 
-# Show the tickets where the subject contains "2"
+```elixir
 Helpdesk.Support.Ticket
 |> Ash.Query.filter(contains(subject, "2"))
 |> Ash.DataLayer.Simple.set_data(tickets)
 |> Helpdesk.Support.read!()
+```
 
-# Show the tickets that are closed and their subject does not contain "4"
+Find the tickets that are _closed_ and their subject does _not_ contain `"4"`
+
+```elixir
 Helpdesk.Support.Ticket
 |> Ash.Query.filter(status == :closed and not(contains(subject, "4")))
 |> Ash.DataLayer.Simple.set_data(tickets)
 |> Helpdesk.Support.read!()
 ```
 
-The examples shown here could be implemented easily using things like `Enum.filter`, but the real power here is to allow you to use the same tools when working with any data layer. If you were using AshPostgres, the above code would be exactly the same, except for the call to `set_data/2`.
+The examples above could be easily implemented with `Enum.filter`, but the real power here is to allow you to use the same tools when working with any data layer. If you were using the (link?) AshPostgres data layer, the above code would be exactly the same, except we wouldn't need the call to `set_data/2`.
+
+> TODO talk about any practical use for Simple Data layer, or explain why it is like it is (so you have something presumably until you set a real data layer)
+
+`Ash.DataLayer.Simple` isn't actually very useful in practice so
 
 ### Adding basic persistence
 
-Before we get into working with relationships, lets add some actual persistence to our resource. This will let us add relationships and try out querying data.
+Before we get into working with relationships, lets add some real persistence to our resource. This will let us add relationships and try out querying data.
 
-There is a built in data layer that is good for testing and prototyping, that uses [ETS](https://elixir-lang.org/getting-started/mix-otp/ets.html).
+There is a built in data layer that is good for testing and prototyping, that uses [ETS](https://elixir-lang.org/getting-started/mix-otp/ets.html). ETS (Erlang Term Storage) is OTP's in-memory database, so the data won't actually stick around beyond the lifespan of your program, but it's simple, and behaves just like a real database.
```

Comment:
I'd hesitate to say "behaves just like a real database" as it has tons of limitations (no transactions, no timeouts, things like that)

Reply:
Fair point